handlers/file-base64-upload.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * file-base64
+ * return a file as base64
+ * our Request handler.
+ */
+const async = require("async");
+const fs = require("fs");
+const path = require("path");
+const PathUtils = require("../utils/pathUtils.js");
+const child_process = require("child_process");
+
+const ABBootstrap = require("../AppBuilder/ABBootstrap");
+// {ABBootstrap}
+// responsible for initializing and returning an {ABFactory} that will work
+// with the current tenant for the incoming request.
+const serviceKey = "file_processor.file-base64-upload"; // this is how listeners will identify this service.
+
+module.exports = {
+   /**
+    * Key: the cote message key we respond to.
+    */
+   key: serviceKey,
+   inputValidation: {
+      // uuid: { string: { uuid: true }, required: true },
+      photoID: { string: { uuid: true }, required: true },
```

Comment:
Can you change this to `fileID` to be consistent with the service key. Might not be a photo.